### PR TITLE
Make #windows? return true / false

### PR DESCRIPTION
**Pull request by JonRowe**
As suggested by @myronmarston in #220

### CODE REVIEW

File: lib/rspec/support/ruby_features.rb [hunk 1]
```diff
@@ -9,11 +9,11 @@ module OS
       module_function
 
       def windows?
-        RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/
+        !!(RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/)
       end
 
       def windows_file_path?
-        ::File::ALT_SEPARATOR == '\\'
+        !!(::File::ALT_SEPARATOR == '\\')
```
Comment (myronmarston) [inline lib/rspec/support/ruby_features.rb]: `String#==` already returns `true` or `false` so the `!!` is unnecessary here.

Comment (JonRowe) [inline lib/rspec/support/ruby_features.rb]: Derp, for some reason I read this as a regexp.

Comment (JonRowe) [inline lib/rspec/support/ruby_features.rb]: Updated

**Comment by myronmarston** [comment]
LGTM.

**Pull request merged**